CHANGELOG.md

```diff
@@ -7,12 +7,9 @@
 - SpringBoot Starter now supports reading iKey using all the variable names as core sdk.
 - Starter would no longer support relaxed binding of ikey property due to complex conditional need and backport problems with RelaxedBinder from Boot 2 to 1.5.x.
 - `InterceptorRegistry` class no longer has `@EnableWebMvc` annotation as it breaks springboot autoconfig.
-- Deprecated `getRoleName`/`setRoleName` and `getRoleInstance`/`setRoleInstance` in `DeviceContext`. 
-Introduced `CloudContext` to hold replacements, `getRole`/`setRole` and `getRoleInstance`/`setRoleInstance`, respectively.
-- Introduced `CloudInfoContextInitializer` to set roleInstance in `CloudContext`.
-`DeviceInfoContextInitializer` will continue to send the deprecated field in `DeviceContext`.
-This will be removed in a future release.
-Users should use begin using the `CloudInfoContextInitializer` to continue to receive the proper value for `cloud_roleInstance` in telemetry items.
+- Deprecated `getRoleName`/`setRoleName` and `getRoleInstance`/`setRoleInstance` in `DeviceContext`. Introduced `CloudContext` to hold replacements, `getRole`/`setRole` and `getRoleInstance`/`setRoleInstance`, respectively. 
+- Introduced `CloudInfoContextInitializer` to set roleInstance in `CloudContext`. `DeviceInfoContextInitializer` will continue to send the deprecated field in `DeviceContext`. This will be removed in a future release. The new initializer is included by default and therefore will not affect the current tags.
```

Comment:
@littleaj I don't think that `DeviceInfoContextInitializer` continues to send the deprecated fields. I think we discussed this and then you made the changes. Here is what the code is like 
```
 @Override
    public void initialize(TelemetryContext context)
    {
        DeviceContext device = context.getDevice();
        device.setOperatingSystem(DeviceInfo.getOperatingSystem());
        device.setOperatingSystemVersion(DeviceInfo.getOperatingSystemVersion());
        device.setId(DeviceInfo.getHostName());
        device.setLocale(DeviceInfo.getLocale());
    }
```
Don't think anything is deprecated here. Could you please fix this release notes.

Reply:
good catch